src/defaults.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function() {
+  return {
+    wallet_server: 'http://localhost:3000/v2',
+
+    kdf_params: {
+      algorithm: 'scrypt',
+      bits: 256,
+      n: 2,
+      r: 2,
+      p: 1
```

Comment:
I wonder if we should leave a comment on the performance / security trade-off used to choose these defaults?